propulate/propagators/pso/pso_init_uniform.py

```diff
@@ -0,0 +1,110 @@
+"""
+This file contains propagators, that can be used to initialize a population of either Individuals or Particles.
+"""
+from random import Random
+from typing import Union, Dict, Tuple, List
+
+import numpy as np
+
+from propulate.particle import Particle
+from propulate.population import Individual
+from propulate.propagators import Stochastic
+from propulate.utils import make_particle
+
+
+class PSOInitUniform(Stochastic):
+    """
+    Initialize individuals by uniformly sampling specified limits for each trait.
+
+    This propagator is the Particle-using counterpart to the InitUniform propagator.
+    For more information, please have a look there.
+    """
+
+    def __init__(
+        self,
+        limits: Dict[str, Tuple[float, float]],
+        parents=0,
+        probability=1.0,
+        rng: Random = None,
+        *,
+        v_init_limit: Union[float, np.ndarray] = 0.1,
+        rank: int
+    ):
+        """
+        Constructor of PSOInitUniform class.
```

Comment:
```suggestion
        Constructor of InitUniformPSO class.
```

Reply:
This is done differently, thus cancelled.